tests/Common/ComponentsTest.php

```diff
@@ -756,75 +756,90 @@ public function testComponentConfigUpdateChangeDescription()
 
     public function testComponentConfigsVersionsList()
```

Comment:
Mohl by si prosím podobně upravit i `testComponentConfigRowVersionsList` ? A potom se  může vyhodit ten `testScenarioFromConnection1256`.

Reply:
Upraveno. Speciálně jsem narazil na [situaci](https://github.com/keboola/storage-api-php-client/blob/tf2-test-connection-1256-scenario/tests/Common/ComponentsTest.php#L1890-1890), kde si myslím, že je to bug (resp. přehlédnutí) a ne cílená feature

Reply:
Myslel jsem to, jak se liší getVersion od toho, co vrátí updateRow... 

created u verze je ve skutečnosti "createdVersion"
id není vůbec

Reply:
jo jasně, šlo tedy o tohle https://keboola.slack.com/archives/C02CGRFH8/p1523430950000060?thread_ts=1523388533.000001&cid=C02CGRFH8

Reply:
jojo